var/spack/repos/builtin/packages/bowtie2/package.py

```diff
@@ -66,4 +66,4 @@ def build_targets(self):
 
     @property
     def install_targets(self):
-        return ['PREFIX={0}'.format(self.prefix), 'install']
+        return ['prefix={0}'.format(self.prefix), 'install']
```

Comment:
```suggestion
        if self.spec.satisfies('@:2.3.9'):
            return ['prefix={0}'.format(self.prefix), 'install']
        else:
            return ['PREFIX={0}'.format(self.prefix), 'install']
```
the Makefile variable is changed from prefix to PREFIX in 2.4.0 beta2 .
https://github.com/BenLangmead/bowtie2/commit/8d123c66c1f637604046add1a15120ad04060ec0

Reply:
@tkameyama -- your change also works for me w/ `@2.3.5.1`.

:satisfied:

Reply:
@tkameyama -- would you like to fix this or would you like me to update this PR to use your patch?

Reply:
Please update to you.

Reply:
done.